src/Database/Adapter/Mongo.php

```diff
@@ -1008,7 +1010,7 @@ protected function getQueryOperator(string $operator): string
                 return '$search';
 
             default:
-                throw new Exception('Unknown Operator:' . $operator);
+                throw new DatabaseException('Unknown Operator:' . $operator);
```

Comment:
```suggestion
                throw new DatabaseException('Unknown Operator:' . $operator . "Must be one of ..... ");
```
Let's make this message more descriptive